openmicroscopy/openmicroscopy
# Download menus web

Updates Download Menu. See https://trello.com/c/3n7Plb19/189-scripts-menus-download-options

To test:
 - Select an image and compare Download options with Insight and trello card above.
 - Also check Download menu with other objects selected P/D and SPW data.
 - Check menu with multiple Images and other objects selected (batch download).

![screen shot 2015-01-27 at 11 43 43](https://cloud.githubusercontent.com/assets/900055/5917870/e0e4428e-a619-11e4-8e9f-c25d98efd457.png)

![screen shot 2015-01-27 at 11 49 35](https://cloud.githubusercontent.com/assets/900055/5917960/ca4c3062-a61a-11e4-9040-d8f3e4a98e0c.png)

 - Also check buttons in the Activities panel after running a script E.g. ThumbnailFigure on an image with a long name. (fixes: https://github.com/openmicroscopy/openmicroscopy/pull/2970#issuecomment-55227394)

![screen shot 2015-01-27 at 11 32 26](https://cloud.githubusercontent.com/assets/900055/5917982/078c30a8-a61b-11e4-8e06-4dbda7e343bf.png)
